nuxeo-rating-core/src/test/java/org/nuxeo/ecm/rating/TestRatingService.java

```diff
@@ -249,7 +251,7 @@ public void shouldNotFindRatingWhenDocIsDeleted() {
         assertEquals(1, ratingService.getRatesCount(docActivity, STARS_ASPECT));
 
         DocumentRef docRef = doc.getRef();
-        session.followTransition(docRef, "delete");
+        Framework.getService(TrashService.class).trashDocument(doc);
         session.save();
```

Comment:
`TrashService` already call the `session#save` method. You can remove this line.